api/swagger.yaml

```diff
@@ -19,18 +19,18 @@ paths:
               schema:
                 $ref: '#/components/schemas/chat_list_response'
               example:
-                - chat_id: 543
-                  user_ids: [123, 234]
-                  last_activity: "2022-10-10T18:29:48.843Z"
+                - chatId: 543
+                  dancerIds: [123, 234]
+                  lastActivity: "2022-10-10T18:29:48.843Z"
                   type: "group"
                   messages: []
```

Comment:
@jans510 and @SteliosRusta 
I think we talked about that we do not want the list message to appear in the collection endpoint of chats and not in the result of the endpoint that creates one new chat, correct?

We considered that because, we anyways will have to invoke the messages endpoint anyways so there is no need to send them twice, righty?

Reply:
yes, it can be removed completely from the list endpoints payload

Reply:
I removed it not only from the list endpoint but also from the chat details endpoints.
I would guess for the same reason ;-) Agree?
(I also followed the way you @jans510 structured your swagger file with the components like schemas and examples :-) )

Reply:
OK, I will drop this. We will achieve those things with an asynchronous api